### DISCUSSION TITLE
Finished Websites on webstudio / Home service industry

### DISCUSSION THREAD

**Discussion by Ivan-ths** [webstudio-is/webstudio-community#156] [General]
Hi does anyone have any finished website we can look at? We are in the home service industry building websites for Junk Removal businesses. Im looking to build large websites that have 100-500 pages on them. How does webstudio hold up with that many pages? Are the speeds capable with that many pages?

**Comment by kof** [comment]
Do you mean 500 custom static pages or cms pages? We have a bunch of templates for cms in the marketplace and a few design templates

**Reply by kof** [reply]
If most of these are cms pages, then webstudio doesn't care, since its basically 1 dynamic template. If its 500 static unique pages  then you would be doing something wrong.

Haveing a couple of dozens of unique custom pages should be fine though.
